libs/blocks/sticky-header/sticky-header.js

```diff
@@ -0,0 +1,24 @@
+import { createTag, getConfig } from '../../utils/utils.js';
+
+export default function decorate($block) {
+  const $header = document.querySelector('header');
+  const $div = $block.firstElementChild;
+  const $a = $block.querySelector('a');
+  const $close = createTag('a', { class: 'sticky-header-close' });
+
+  const { miloLibs, codeRoot } = getConfig();
+  const base = miloLibs || codeRoot;
+  const $closeIcon = createTag('img', { class: 'sticky-header-close-icon', src: `${base}/blocks/sticky-header/close.svg` });
+
+  $header.parentNode.insertBefore($block, $header.nextElementSibling);
+
+  $a.classList.add('button');
+  $a.target = '_blank';
```

Comment:
if you want a link to open in a new tab just add #_blank to the end of the link. This doesn't need to be in the code.

Reply:
Interesting point. I wasn't aware of it.